mmdet/core/hook/__init__.py

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 from .checkloss_hook import CheckInvalidLossHook
 from .ema import ExpMomentumEMAHook, LinearMomentumEMAHook
+from .memory_profiler_hook import MemoryProfilerHook
```

Comment:
Would it be more appropriate to put `mmcv` directly in this code? Because subsequent other repo also need

Reply:
That's true. We should consider to put it in mmcv.